optimize.py

```diff
@@ -195,13 +198,74 @@ def _check_missing_players(all_players, min_cost, e_raise):
                                                 str(total_report))
         raise Exception('Total missing players at price point: ' + str(miss))
 
+def create_upload_file_and_map_pids(pid_file, test_mode=False):
+    # create upload file
+    csv_name = 'test' if test_mode else 'current'
+    with open(fnu.format(csv_name), "w") as f:
+        writer = csv.writer(f)
+        writer.writerow(['QB','RB','RB', 'WR', 'WR', 'WR', 'TE', 'FLEX', 'DST'])
+
+    # create a player map from name/position to id
+    player_map = {}
+    with open(pid_file, "r") as f:
+        n = 0
+        for line in f.readlines():
+            n += 1
+            if 'TeamAbbrev' in line: # line with field names was found
+                fields= line.split(',')
+                break
+        f.close()
+        f = islice(open(pid_file, "r"), n, None)
+        reader = csv.DictReader(f, fieldnames=fields)
```

Comment:
In general, we need more in the README or some type of other instruction about what you need to download to run this. DraftKings calls both the normal CSV download DKSalaries.csv and the template option from https://www.draftkings.com/lineup/upload the same thing, even though these CSVs are structured totally differently.